extensions/cornerstone/src/initDoubleClick.ts

```diff
@@ -0,0 +1,59 @@
+import { eventTarget, EVENTS } from '@cornerstonejs/core';
+import { Enums } from '@cornerstonejs/tools';
+import { CommandsManager, CustomizationService, Types } from '@ohif/core';
+
+const cs3DToolsEvents = Enums.Events;
+
+const DEFAULT_DOUBLE_CLICK_COMMAND: Types.Command = {
+  commandName: 'toggleOneUp',
+  commandOptions: {},
+};
+
+export type initDoubleClickArgs = {
+  customizationService: CustomizationService;
+  commandsManager: CommandsManager;
+};
+
+function initDoubleClick({
+  customizationService,
+  commandsManager,
+}: initDoubleClickArgs): void {
+  const cornerstoneViewportHandleDoubleClick = _ => {
+    const customizations: Types.Command | Types.CommandCustomization =
+      (customizationService.get(
+        'cornerstoneViewportDoubleClickCommands'
+      ) as Types.CommandCustomization) || DEFAULT_DOUBLE_CLICK_COMMAND;
+
+    commandsManager.run(customizations);
+  };
+
+  function elementEnabledHandler(evt: CustomEvent) {
```

Comment:
also we should run getNearbyToolData command, just before we run ours

Reply:
Got it.